packages/bpk-tokens/tokens/base.raw.android.json

```diff
@@ -309,1108 +309,848 @@
     },
     "TEXT_SM_FONT_WEIGHT": {
       "value": "400",
       "type": "string",
       "category": "font-weights",
-      ".alias": {
-        "value": "400"
-      },
+      "originalValue": "{!FONT_WEIGHT_REGULAR}",
       "name": "TEXT_SM_FONT_WEIGHT"
     },
     "TEXT_SM_LINE_HEIGHT": {
       "value": "20",
       "type": "size",
       "category": "line-heights",
-      "meta": {
-        "deprecated": true
-      },
-      ".alias": {
-        "value": 20
-      },
-      "name": "TEXT_SM_LINE_HEIGHT"
+      "name": "TEXT_SM_LINE_HEIGHT",
+      "originalValue": "{!LINE_HEIGHT_SM}"
     },
     "TEXT_XL_FONT_SIZE": {
       "value": "24",
       "type": "font-size",
       "category": "font-sizes",
-      ".alias": {
-        "value": 24
-      },
+      "originalValue": "{!FONT_SIZE_XL}",
       "name": "TEXT_XL_FONT_SIZE"
     },
     "TEXT_XL_FONT_WEIGHT": {
       "value": "400",
       "type": "string",
       "category": "font-weights",
-      ".alias": {
-        "value": "400"
-      },
+      "originalValue": "{!FONT_WEIGHT_REGULAR}",
       "name": "TEXT_XL_FONT_WEIGHT"
     },
     "TEXT_XL_LINE_HEIGHT": {
       "value": "32",
       "type": "size",
       "category": "line-heights",
-      "meta": {
-        "deprecated": true
-      },
-      ".alias": {
-        "value": 32
-      },
-      "name": "TEXT_XL_LINE_HEIGHT"
+      "name": "TEXT_XL_LINE_HEIGHT",
+      "originalValue": "{!LINE_HEIGHT_XL}"
     },
     "TEXT_XS_FONT_SIZE": {
       "value": "12",
       "type": "font-size",
       "category": "font-sizes",
-      ".alias": {
-        "value": 12
-      },
+      "originalValue": "{!FONT_SIZE_XS}",
       "name": "TEXT_XS_FONT_SIZE"
     },
     "TEXT_XS_FONT_WEIGHT": {
       "value": "400",
       "type": "string",
       "category": "font-weights",
-      ".alias": {
-        "value": "400"
-      },
+      "originalValue": "{!FONT_WEIGHT_REGULAR}",
       "name": "TEXT_XS_FONT_WEIGHT"
     },
     "TEXT_XS_LINE_HEIGHT": {
       "value": "18",
       "type": "size",
       "category": "line-heights",
-      "meta": {
-        "deprecated": true
-      },
-      ".alias": {
-        "value": 18
-      },
-      "name": "TEXT_XS_LINE_HEIGHT"
+      "name": "TEXT_XS_LINE_HEIGHT",
+      "originalValue": "{!LINE_HEIGHT_XS}"
     },
     "TEXT_XXL_FONT_SIZE": {
       "value": "34",
       "type": "font-size",
       "category": "font-sizes",
-      ".alias": {
-        "value": 34
-      },
+      "originalValue": "{!FONT_SIZE_XXL}",
       "name": "TEXT_XXL_FONT_SIZE"
     },
     "TEXT_XXL_FONT_WEIGHT": {
       "value": "400",
       "type": "string",
       "category": "font-weights",
-      ".alias": {
-        "value": "400"
-      },
+      "originalValue": "{!FONT_WEIGHT_REGULAR}",
       "name": "TEXT_XXL_FONT_WEIGHT"
     },
     "TEXT_XXL_LINE_HEIGHT": {
       "value": "40",
       "type": "size",
       "category": "line-heights",
-      "meta": {
-        "deprecated": true
```

Comment:
This is also gone. I wouldn't think anyone relies on this metadata though, it's more for us in the source